pkg/antctl/raw/multicluster/common/cleanup.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Common codes for both leader and member cluster to clean up the Antrea
+// Multi-cluster resources in the given Namespace.
+
+package common
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+type CleanOptions struct {
+	Namespace  string
+	ClusterSet string
+}
+
+func (o *CleanOptions) validate() error {
+	if o.ClusterSet == "" {
+		return fmt.Errorf("ClusterSet is required")
+	}
+
+	return nil
+}
+
+func Cleanup(cmd *cobra.Command, cleanOpts *CleanOptions) error {
```

Comment:
We should also delete the Secrete created by init/join, and the SA and RB created by init.

Reply:
Or for simplicity, we just delete all resources with the "createdByAntctl" annotation.

Reply:
Now the command will delete all resources with "createdByAntctl" annotation.